test/vectors/issuance/2.issue-credential_offer.json

```diff
@@ -0,0 +1,198 @@
+{
+   "type":"https://didcomm.org/issue-credential/3.0/offer-credential",
```

Comment:
this URL is 404, I suggest opening an issue on that repo to track fixes for this, it does not need to be solved to merge, but it does need to be tracked.

Reply:
Updated the issue - https://github.com/decentralized-identity/waci-didcomm/issues/4